hr_expense_cancel/views/hr_expense_views.xml

```diff
@@ -2,7 +2,6 @@
 <!-- Copyright 2019 Tecnativa - Ernesto Tejeda
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 <odoo>
-
```

Comment:
Why removing these lines?

Reply:
Fixed in [commit](https://github.com/OCA/hr/pull/564/commits/f4db176c4f844ed47892a5cb584695efc5573fc8)

Reply:
Fixed in [commit](https://github.com/OCA/hr/pull/564/commits/f4db176c4f844ed47892a5cb584695efc5573fc8)